### DISCUSSION TITLE
Unsupported video mode

### DISCUSSION THREAD

**Discussion by abrusco** [eduardofilo/RG350_adam_image#84] [RetroArch topics]
![20210822_123741_001](https://user-images.githubusercontent.com/8458577/130361432-c2f369f6-79a5-4117-993b-73352137fb59.jpg)
In the mame, even making the recommended changes to the wiki, In some games like mortal kombat the error continues. 
Emulator mame2003
![20210822_124010](https://user-images.githubusercontent.com/8458577/130361422-06f10a2e-c9d0-4684-b02f-f9f3d1c6f8df.jpg)
![20210822_123924](https://user-images.githubusercontent.com/8458577/130361423-b2a61912-4b32-4538-8eae-31e09cdf8738.jpg)

**Comment by eduardofilo** [comment]
It is not a exact rule. In that cases, try with one of the standalone emulators. That game runs on FBA, but slow. Maybe is too much heavy for this consoles. Press `Select` on the SimpleMenu list of ROMs and change `Emulator` to `fba-multi.opk`.
